rclcpp/include/rclcpp/experimental/buffers/ring_buffer_implementation.hpp

```diff
@@ -67,6 +70,7 @@ class RingBufferImplementation : public BufferImplementationBase<BufferT>
 
     write_index_ = next_(write_index_);
     ring_buffer_[write_index_] = std::move(request);
+    TRACEPOINT(ring_buffer_enqueue, static_cast<const void *>(this), write_index_, size_+1, is_full_());
```

Comment:
```suggestion
    TRACEPOINT(
      ring_buffer_enqueue,
      static_cast<const void *>(this),
      write_index_,
      size_ + 1,
      is_full_());
```

Reply:
fixed in [9ead575](https://github.com/ros2/rclcpp/pull/2091/commits/9ead5753751ed694c7acac47483c6f37ef61919d).